apps/finance/contracts/Finance.sol

```diff
@@ -104,24 +102,21 @@ contract Finance is AragonApp, ERC677Receiver {
      *      to Vault.
      * @notice Allows to send ETH from this contract to Vault, to avoid locking them in contract forever.
      */
-    function escapeHatch() public payable {
-        // convert ETH to EtherToken
-        etherToken.wrapAndCall.value(this.balance)(address(this), "Adding Funds");
+    function () public payable {
+        vault.deposit.value(this.balance)(ETH, msg.sender, this.balance, new bytes(0));
```

Comment:
Somewhat confused about this—do we want this to be the fallback (rather than an explicit "escape hatch")?

Also, should it be `this` and not `msg.sender`, e.g.:

`vault.deposit.value(this.balance)(ETH, this, this.balance, new bytes(0));`

Reply:
Pinging @izqui, who came up with this change [earlier](https://github.com/aragon/aragon-apps/pull/135#discussion_r175007166).